sharding-sql-test/src/main/resources/sql/supported/ddl/truncate.xml

```diff
@@ -17,7 +17,7 @@
   -->
 
 <sql-cases>
-    <sql-case id="truncate_table" value="TRUNCATE TABLE t_log" />
+    <sql-case id="truncate_table" value="TRUNCATE TABLE t_log" db-types="MySQL,Oracle,PostgreSQL"/>
```

Comment:
Hi, why did you want to limit to those databases?

Reply:
This testcase for H2 failed, so remove it. I just change mysql parser . I think this change will not affect others.

Reply:
Why did H2 fail? AFAIK, when db is h2, parser test engine will use MySQL to parse SQL.

Reply:
Not parser test,it is jdbc-test.